artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/TransportConstants.java

```diff
@@ -183,6 +187,10 @@
 
    public static final int DEFAULT_TCP_RECEIVEBUFFER_SIZE = 1024 * 1024;
 
+   public static final int DEFAULT_NETTY_WRITE_BUFFER_LOW_WATER_MARK = 32 * 1024;
```

Comment:
This clashes with DEFAULT_TCP_RECEIVERBUFFER_SIZE, doesn't? 

We need to put this value down though.